src/components/input-time-picker/input-time-picker.e2e.ts

```diff
@@ -294,25 +356,160 @@ describe("calcite-input-time-picker", () => {
+    await page.waitForChanges();
+
+    expect(changeEvent).toHaveReceivedEventTimes(1);
+    expect(await inputTimePicker.getProperty("value")).toBe("17:04:00");
+    expect(await getInputValue(page)).toBe("05:04:00 PM");
+
+    await page.keyboard.press("Enter");
+    await page.waitForChanges();
+
+    expect(changeEvent).toHaveReceivedEventTimes(1);
+  });
+
+  it("committing hh:mm value when step=1 with the Enter key works as expected for a 24-hour locale", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`<calcite-input-time-picker step="1" lang="fr"></calcite-input-time-picker>`);
+
+    const inputTimePicker = await page.find("calcite-input-time-picker");
+    const changeEvent = await inputTimePicker.spyOnEvent("calciteInputTimePickerChange");
+
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+
+    await page.keyboard.press("Tab");
+    await page.keyboard.type("14:2");
+    await page.waitForChanges();
 
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+
+    await page.keyboard.press("Enter");
+    await page.waitForChanges();
+
+    expect(changeEvent).toHaveReceivedEventTimes(1);
+    expect(await inputTimePicker.getProperty("value")).toBe("14:02:00");
+    expect(await getInputValue(page)).toBe("14:02:00");
+
+    await page.keyboard.press("Enter");
     await page.waitForChanges();
 
     expect(changeEvent).toHaveReceivedEventTimes(1);
+  });
+
+  it("attempting to commit an invalid time value fails, but leaves the typed value intact", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`<calcite-input-time-picker step="1"></calcite-input-time-picker>`);
 
-    await page.keyboard.press(":");
-    await page.keyboard.press("3");
+    const inputTimePicker = await page.find("calcite-input-time-picker");
+    const changeEvent = await inputTimePicker.spyOnEvent("calciteInputTimePickerChange");
+
+    expect(changeEvent).toHaveReceivedEventTimes(0);
 
+    await page.keyboard.press("Tab");
+    await page.keyboard.type("26:0:0");
     await page.waitForChanges();
 
-    expect(changeEvent).toHaveReceivedEventTimes(2);
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+
+    await page.keyboard.press("Enter");
+    await page.waitForChanges();
+
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+    expect(await inputTimePicker.getProperty("value")).toBe("");
+    expect(await getInputValue(page)).toBe("26:0:0");
+
+    await page.keyboard.press("Enter");
+    await page.waitForChanges();
+
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+
+    await page.keyboard.press("Tab");
+
+    expect(changeEvent).toHaveReceivedEventTimes(0);
+    expect(await inputTimePicker.getProperty("value")).toBe("");
+    expect(await getInputValue(page)).toBe("26:0:0");
   });
 
-  it("formats valid typed time value appropriately on blur", async () => {
+  it("blurring the input with a valid time value works as expected for 12-hour locale", async () => {
```

Comment:
For readability, should this mention that the value is committed vs "works as expected"?

Applies to similar test names.